src/java.desktop/share/classes/javax/swing/text/html/HTMLEditorKit.java

```diff
@@ -1307,6 +1299,27 @@ public void handleEndOfLineString(String eol) {
      *     <td>FrameView
      * </tbody>
      * </table>
+     *
+     * <p>
+     * @implNote
```

Comment:
The `<p>` tag before `@implNote` is unneeded, javadoc will generate a section code.

At least I can't see it used in other places. I'm currently building docs with this change to confirm.

Reply:
> The `<p>` tag before `@implNote` is unneeded, javadoc will generate a section code.

It must be removed, it results in `make docs` failure:

```
=== Output from failing command(s) repeated here ===
* For target jdk_modules_java.desktop__the.java.desktop_batch:
...\classes\javax\swing\text\html\HTMLEditorKit.java:1303:
warning: empty <p> tag
     * <p>
       ^
```